lib/LedgerSMB/Template/HTML.pm

```diff
@@ -34,6 +34,8 @@ sub escape {
     return undef unless defined $vars;
     #$vars = encode_entities($vars);
     $vars = escape_html($vars);
+    #Allow back only a few decorations.
+    $vars =~ s~&lt;(/?[biu]|br)&gt;~<$1>~g;
```

Comment:
This is the wrong place to do that. What if someone added <b> to their invoice comments for some dark reason?

A better place to put the decoding is in the template itself, where it can be a specific contract between the string provider and the template, for specifically only those strings.

The HTML template sets up a function named `UNESCAPE` (https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Template/HTML.pm#L54) to allow the template to reverse escaping of specific strings.